hmcts/bulk-scan-shared-infrastructure
# Added functionality to spin up ccd and import definitions locally

### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/RPE-699

### Change description ###

- Added functionality to spin up CCD stack with all dependent services locally.

- Added scripts to import CCD Definitions and utilities related to Idam to create roles, users  etc.

- Tested locally

- TODO

1. Publish docker image for bulk scan orchestrator(enableDockerBuild and few other config).
2. Update Readme with ccd docker info in shared infra.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x ] No
```
